packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -295,6 +297,10 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
     private renderCustomerStep(step: CheckoutStepStatus): ReactNode {
         const { isGuestEnabled } = this.props;
 
+        const updateStripeLinkAuthenticated = (auth: boolean = false) => {
+            this.setState({ isStripeLinkAuthenticated: auth });
```

Comment:
hey @AlanMunozN this seems ok, but after another look this feels like can be moved to sdk state. Since I can see we use this in payments step below. And possibly in shipping step in next PR as well.

Reply:
Hi @animesh1987, do you mean something like this? https://github.com/bigcommerce/checkout-sdk-js/compare/master...AlanMunozN:checkout-sdk-js:STRIPE-52-peng

Reply:
@AlanMunozN Appreciate your effort. Now we can use checkout context to get `isStripeLinkAuthenticated` from SDK directly, no need to set an addtional prop here. You simply need:
1. import context:
`import checkoutContext from "../../checkout/CheckoutContext";`
2. Use context to get checkout state, for example, getting the value of feature flag `CHECKOUT-3790.upgrade_checkout_payment_step`:
```
const context = useContext(checkoutContext);
const flagValue = context?.checkoutState.data.getConfig()?.checkoutSettings?.features['CHECKOUT-3790.upgrade_checkout_payment_step']
```

Reply:
@bc-peng, we'll wait for the resolution of the preview comment because could affect the complete approach